### PR TITLE
added changes to enable cloud watch logs in query workbench

### PR DESCRIPTION
**Pull request by sumukhswamy**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by ps48 — CHANGES_REQUESTED** [review]
cannot merge disabling options to main

File: common/constants/index.ts [hunk 1]
```diff
@@ -19,11 +19,7 @@ export const TREE_ITEM_LOAD_MATERIALIZED_BADGE_NAME = `Load Materialized View`;
 export const TREE_ITEM_BADGE_NAME = `badge`;
 export const LOAD_OPENSEARCH_INDICES_QUERY = `SHOW tables LIKE '%';`;
 export const SKIPPING_INDEX_QUERY = `CREATE SKIPPING INDEX ON datasource.database.table 
-(status VALUE_SET) 
-WITH (
-  auto_refresh = true,
-  checkpoint_location = 's3://test/'
-  )`;
+(status VALUE_SET) `;
```
Comment (ps48) [inline common/constants/index.ts:22]: need to add auto refresh as false
```
WITH (
  auto_refresh = false
  )`;

Comment (ps48) [reply common/constants/index.ts:22]: Please check with Sean on this.

File: public/components/SQLPage/DataSelect.tsx [hunk 2]
```diff
@@ -29,12 +29,13 @@ export const DataSelect = ({ http, onSelect, urlDataSource, asyncLoading }: Cust
         const data = res.data.resp;
 
         const connectorGroups = {};
+        console.log(data)
```
Comment (ps48) [inline public/components/SQLPage/DataSelect.tsx:32]: can remove this

File: public/components/SQLPage/DataSelect.tsx [hunk 3]
```diff
@@ -29,12 +29,13 @@ export const DataSelect = ({ http, onSelect, urlDataSource, asyncLoading }: Cust
...
 
         data.forEach((item) => {
           const connector = item.connector;
           const name = item.name;
 
-          if (connector === 'S3GLUE') {
+          if (connector === 'S3GLUE' || connector === 'cloudwatchlog') {
```
Comment (ps48) [inline public/components/SQLPage/DataSelect.tsx:38]: let's convert to lowercase and check.

File: public/components/SQLPage/table_view.tsx [hunk 4]
```diff
@@ -443,6 +443,7 @@ export const TableView = ({ http, selectedItems, updateSQLQueries, refreshTree }
                         if (table.name === tableName) {
                           return {
                             ...table,
+                            isLoading: false,
```
Comment (ps48) [inline public/components/SQLPage/table_view.tsx:446]: do we need this?

Comment (sumukhswamy) [reply public/components/SQLPage/table_view.tsx:446]: no wil remove

**Pull request closed without merging**